source/common/common/backoff_strategy.cc

```diff
@@ -29,8 +29,9 @@ JitteredLowerBoundBackOffStrategy::JitteredLowerBoundBackOffStrategy(
     : min_interval_(min_interval), random_(random) {}
 
 uint64_t JitteredLowerBoundBackOffStrategy::nextBackOffMs() {
-  // random(min_interval_, 1.5 * min_interval_)
-  return (random_.random() % (min_interval_ >> 1)) + min_interval_;
+  // random(min_interval, 1.5 * min_interval)
+  uint64_t min_interval_non_zero = std::max(min_interval_ >> 1, 1UL);
```

Comment:
Sorry how does this happen? This seems like maybe a config validation issue? Should we fix that and have an assert here? cc @asraa for potential fuzzing.

Reply:
Yeah, an assert makes more sense indeed.
I combed through the git history and there was an analogous fix made for the other backoff strategy in the past: https://github.com/envoyproxy/envoy/commit/da98b80fc260702b90f6f49a3967883b4bf59221

It occurred to me that it would be possible to send a reset header with the value 0 that would trigger this code path.

Reply:
Thanks for flagging this! This hasn't come up in config fuzzing yet, but thanks for flagging the reset header codepath. This can be covered in router fuzzing, it's a WIP tracked here https://github.com/envoyproxy/envoy/issues/12568

Reply:
FWIW I think this bug would have been an issue with an untrusted upstream threat model so definitely worth looking at coverage, thanks for flagging.